src/main/scala/com.socrata.tileserver/util/GeoResponse.scala

```diff
@@ -28,12 +31,12 @@ trait GeoResponse extends ResponseInfo {
   /** The (binary) payload from the underlying response. */
   def payload: Array[Byte]
 
-  /** The unpacked features without any processing. */
+
+/** The unpacked features without any processing. */
```

Comment:
It looks like some indentation was accidentally deleted here and an extra newline was added.